lib/rules/web/ocm_console/install_page.xyaml

```diff
@@ -424,21 +428,18 @@ check_gcp_upi_install_page:
     action: check_install_page_common_part_two
     action: check_data_collection_message_in_install_page
     action: check_install_page_part_three
+    action: check_install_page_rhcos_part
     elements:
         - selector:
             xpath: //h2[text()='Follow the documentation to configure your GCP account and run the installer']
         - selector:
             xpath: //p[text()='The installer will take about 45 minutes to run.']
         - selector:
             xpath: //a[contains(@href, 'installing_gcp/installing-gcp-user-infra.html') and text()='Get started']
-        - selector:
-            xpath: //h3[text()='Red Hat Enterprise Linux CoreOS (RHCOS)']
-        - selector:
-            xpath: //p[text()='Download RHCOS to create machines for your cluster to use during installation.']
         - selector:
             xpath: //a[contains(@href, '/installing-gcp-user-infra.html#installation-gcp-user-infra-rhcos_installing-gcp-user-infra') and text()='Learn more']
         - selector:
-            xpath: //a[contains(@href, 'https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/latest/latest/') and text()='Download RHCOS']
+            xpath: //a[contains(@href, '/rhcos/latest/latest/rhcos-4.5.6-x86_64-gcp.x86_64.tar.gz') and text()='Download RHCOS tar']
```

Comment:
Is it possible to use regular expression or something like that to avoid specific version info here, or else we have to update the version again and again ?

Reply:
Not a blocker. Can be optimized in another PR.